servers/quarkus-server/build.gradle.kts

```diff
@@ -156,6 +156,13 @@ tasks.withType<Test>().configureEach {
   systemProperty("quarkus.smallrye.jwt.enabled", "true")
   // TODO requires adjusting the tests - systemProperty("quarkus.http.test-port", "0") -  set this
   //  property in application.properties
+  systemProperty(
+    "it.nessie.container.postgres.tag",
+    System.getProperty(
```

Comment:
Good catch. This change does allow overriding. 

verified like this
`:nessie-quarkus:intTest --tests "org.projectnessie.server.ITRestApiPostgres" -Dit.nessie.container.postgres.tag="502"`